fsw/src/fm_cmd_utils.h

```diff
@@ -346,7 +346,7 @@ bool FM_VerifyChildTask(uint32 EventID, const char *CmdText);
  *       available entry in the child task queue with the arguments for
  *       the current command.  The function updates the queue access
  *       index and then verifies that the Child Task is operational.
- *       If the Child Task is operational then it is signaled via
+ *       If the Child Task is operational then it is signalled via
```

Comment:
Signaled is actually the correct spelling (in American English; British English uses 2 'l's)